src/server_manager/web_app/app.ts

```diff
@@ -353,8 +353,8 @@ export class App {
         id: this.digitalOceanAccount.getId(),
         name: await this.digitalOceanAccount.getName(),
       };
-      const status = await this.digitalOceanAccount.getStatus();
-      if (status !== digitalocean.Status.ACTIVE) {
```

Comment:
Perhaps add a TODO to show some sort of warning notification on the account entry/UI.

Reply:
This seemed like a good idea so I just did it.